addon/components/paper-input.js

```diff
@@ -104,55 +103,62 @@ export default Component.extend(FocusableMixin, ColorMixin, ChildMixin, Validati
   willDestroyElement() {
     this._super(...arguments);
     if (this.get('textarea')) {
-      $(window).off(`resize.${this.elementId}`);
+      window.removeEventListener('resize', run.bind(this, this.growTextarea));
```

Comment:
I think that this `removeEventListener` is not working as expected, because `run.bind` returns a new function, different from the one used on the previous `window.addEventListener`.

I think we need to do the same trick that you did on #995, which is basically:

```js
this._growTextarea = run.bind(this, this.growTextarea);
window.addEventListener('resize', this._growTextarea);
```

and later

```js
window.removeEventListener('resize', this._growTextarea);
```